### DISCUSSION TITLE
Pre-select rows in ui.table

### DISCUSSION THREAD

**Discussion by Heindl7** [marimo-team/marimo#1995] [Q&A]
Is there a possibility to pre-select rows (or even all rows) for the ui.table ?
So when I start marimo that already some or all elements/ rows are selected

**Comment by mscolnick** [comment]
Not currently, we do allow passing in `value=..` for most UI elements, but have not implemented for `mo.ui.table`

**Reply by konosubakonoakua** [reply]
> Not currently, we do allow passing in `value=..` for most UI elements, but have not implemented for `mo.ui.table`

Hi, @mscolnick.
Can we use index to pre-select values? Sometimes the value of the UI elements will change due to other UI elements or global variables.

**Reply by mscolnick** [reply]
It is not today, but that sounds like good solution. We would accept a contribution to add this change.
